docs/references/environment-variables.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Environment Variables
+toc_order: 2
+---
+
+# Environment Variables
+
+## Kontena Master
+
+- `MONGODB_URI`: MongoDB connection uri (required)
+- `VAULT_KEY`: secret key for the Kontena Vault (required)
+- `VAULT_IV`: initialization vector for the Kontena Vault (required)
+- `WEB_CONCURRENCY`: number of forked master api processes (default: 1)
+- `MAX_THREADS`: number of threads inside single master api process (default: 8)
```

Comment:
I agree.. but master should check number of cpus and adjust this value automatically if user has not set env. (celluloid already has this kind of helper so we could use it)